operator/backup/operator.go

```diff
@@ -12,17 +23,90 @@ type Operator interface {
 }
 
 type backupOperator struct {
-	clientset kubernetes.Interface
+	clientset    kubernetes.Interface
+	verClientset versioned.Interface
 }
 
 // NewBackupOperator returns a new Operator.
-func NewBackupOperator(clientset kubernetes.Interface) Operator {
+func NewBackupOperator(verClientset versioned.Interface, clientset kubernetes.Interface) Operator {
 	return &backupOperator{
-		clientset: clientset,
+		clientset:    clientset,
+		verClientset: verClientset,
 	}
 }
 
 func (b *backupOperator) ScheduleBackup(backup *crv1.MySQLBackup) error {
-	//TODO: implement backup scheduling
+	clusterName := backup.Spec.Cluster
+	_, err := b.verClientset.CrV1().MySQLClusters(metav1.NamespaceDefault).Get(clusterName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	logging.LogBackup(backup).Debug("Creating PVC.")
+	err = b.createPVC(backup)
+	if err != nil {
+		return err
+	}
+
+	logging.LogBackup(backup).Debug("Creating cron job.")
+	err = b.createCronJob(backup)
+	if err != nil {
+		// Cleanup - remove already created PVC
+		logging.LogBackup(backup).WithField("error", err).Warn("Reverting PVC creation.")
+		b.removePVC(backup)
```

Comment:
Like in `Cluster operator` if an error during removal occurs we want to let the user know:

```go
removeErr := removePVC(cluster)
return errors.NewAggregate([]error{err, removeErr})
```